webview/src/plots/components/comparisonTable/cell/ComparisonTableCell.tsx

```diff
@@ -35,10 +49,10 @@ export const ComparisonTableCell: React.FC<{
       onClick={() => zoomPlot(plotImg.url)}
       data-testid="image-plot-button"
     >
-      {plotImg.url && plotImg.classes ? (
+      {plotImg.url && classes ? (
         <ComparisonTableBoundingBoxImg
           src={plotImg.url}
-          classes={plotImg.classes}
+          classes={classes}
```

Comment:
Why not select the classes inside of `ComparisonTableBoundingBoxImg` to avoid passing this as a prop? It can always return `null` if there are no classes.

Reply:
Good point! Updated